src/app/components/ConnectorDrawer/ConnectorDrawer.tsx

```diff
@@ -72,21 +60,12 @@ export const ConnectorDrawer: FunctionComponent<ConnectorDrawerProps> = ({
       <DrawerContent
         panelContent={
           connector ? (
-            <ConnectorDrawerPanelContent
-              name={connector.name}
-              id={connector.id!}
-              bootstrapServer={connector.kafka!.url!}
-              kafkaId={connector.kafka.id}
-              owner={connector.owner!}
-              namespaceId={connector.namespace_id!}
-              createdAt={new Date(connector.created_at!)}
-              modifiedAt={new Date(connector.modified_at!)}
-              status={connector.status?.state!}
-              error={connector.status?.error}
+            <ConnectorDrawerComponent
+              currentConnectorRef={currentConnectorRef}
+              connectorData={connector}
```

Comment:
I think just `connector` would be fine for the property name.  But then wouldn't just the ref from the state machine be enough to pass through this component?  Passing both of these objects strikes me as redundant, it'd probably be better to keep it simpler by passing only the connector instance or the state machine reference.

Reply:
I guess the original implementation was with "connector" and later the state machine was used to get the latest connector data from pooling to update the connector status accordingly.